### ISSUE TITLE
Is this project still in development?

### ISSUE THREAD

**Issue by adamtharani** [deliverybot/deliverybot#67]

**Comment by fentonfentonfenton** [comment]
Yeah, we seem to be missing `kubernetes` base in the folder it's supposed to be in, is this dead?

thanks!

**Comment by crohr** [comment]
For anyone that ends up on this thread, you might be interested in https://github.com/pullpreview/action
